wavelink/tracks.py

```diff
@@ -179,7 +182,8 @@ async def search(
         if node is MISSING:
             node = NodePool.get_node()
 
-        tracks = await node.get_tracks(cls, f"{cls._search_type}:{query}")
+        search = f"{cls._search_type}:" if not URL_REGEX.match(query) else ""
```

Comment:
Wouldn't it be better to add url as a cls search type? And if you are doing it like this please reflect it in the doc string too